app/containers/LoginServices/interfaces.ts

```diff
@@ -0,0 +1,37 @@
+export interface IOpenOAuth {
+	url: string;
+	ssoToken?: string;
+	authType?: string;
+}
+
+export interface IItemService {
+	_id: string;
+	name: string;
+	service: string;
+	authType: string;
```

Comment:
You know the available types.
We could benefit from having them defined here.

```suggestion
	authType: 'facebook' | 'github' | ...;
```